packages/lesswrong/server/editor/conversionUtils.ts

```diff
@@ -275,30 +275,50 @@ export function markdownToHtmlNoLaTeX(markdown: string): string {
   return trimLeadingAndTrailingWhiteSpace(renderedMarkdown)
 }
 
-export async function markdownToHtml(markdown: string): Promise<string> {
+export async function markdownToHtml(markdown: string, options?: {
+  skipMathjax?: boolean
+}): Promise<string> {
   const html = markdownToHtmlNoLaTeX(markdown)
-  return await mjPagePromise(html, trimLatexAndAddCSS)
+  if (options?.skipMathjax) {
+    return html;
+  } else {
+    return await mjPagePromise(html, trimLatexAndAddCSS)
+  }
 }
 
-export async function ckEditorMarkupToHtml(markup: string): Promise<string> {
+async function ckEditorMarkupToHtml(markup: string, skipMathjax?: boolean): Promise<string> {
```

Comment:
nit: would be nice for this arg to match the one in `markdownToHtml` (i.e. be an options object, or vice-versa), unless there's a reason for them to be different